backend/de.metas.business/src/main/java/de/metas/inventory/IInventoryDAO.java

```diff
@@ -49,5 +55,9 @@ default List<I_M_InventoryLine> retrieveLinesForInventoryId(final InventoryId in
 
 	void setInventoryLinesProcessed(InventoryId inventoryId, boolean processed);
 
+	Set<ProductId> retrieveUsedProductsByInventoryIds(@NonNull Collection<Integer> invetoryIds);
+
+	Timestamp retrieveMinInvetoryDateFromSelection(@NonNull Collection<Integer> invetoryIds);
```

Comment:
* pls use InventoryId instead of Integer
* fix typo in `invetoryIds`
* consider returning Instant instead of Timestamp